VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddPostFragment.java

```diff
@@ -122,6 +123,13 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
 
         getTopicList();
 
+        topicsSpinner.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View v, MotionEvent event) {
+                SoftKeyboardUtil.hide(getActivity());
+                return false;
```

Comment:
Calling this method on every touch event is extremely inefficient. You actually want to use a click listener here, but adapter views don't allow those. Probably the right solution would to subclass `AppCompatSpinner`, override the `performClick()` method, and hide the keyboard if the `super` implementation returns `true`. This would have a the added benefit of making this solution generic as part of the widget, instead of having to duplicate this code in each module where this situation applies (though this seems to be the only one currently).